src/SWP/Bundle/TemplateEngineBundle/Service/ContainerService.php

```diff
@@ -138,4 +149,36 @@ public function createNewContainer($name, array $parameters = [])
 
         return $containerEntity;
     }
+
+    private function createNewMenuWidget()
```

Comment:
Ahis should be in NavigationService or something like that - not here. And i don't see sense of this method. Creating menu already have api, this same for creating widgets. If you want this in demo theme then push it to fixtures.